server/queue.php

```diff
@@ -1,23 +1,33 @@
 <?php
 	$root = realpath($_SERVER["DOCUMENT_ROOT"]);
 	require_once "$root/server/config.php";
+	require_once "$root/server/user.class.php";
+	require_once "$root/server/utilities/dateTimezoneUtilities.php";
 	$conn = $DB_CONN;
 
 	// TODO make this handle multiple locations, if necessary
-	$stmt = $conn->prepare("SELECT DISTINCT appointmentId, scheduledTime, firstName, lastName
+	$stmt = $conn->prepare("SELECT Appointment.appointmentId, scheduledTime,
+		firstName, lastName, timeIn, timeReturnedPapers,
+		timeAppointmentStarted, timeAppointmentEnded, completed, notCompletedDescription
```

Comment:
Quick question on this stuff: Why is it necessary that for the queue we pass back `completed` and `notCompletedDescription`? The queue displays the upcoming appointments, I don't think we need to send this data to the front-end, especially for the queue. Furthermore, if an appointment is completed (or notCompleted), then it isn't even shown on the queue, right? I think it would be best to take these things out.

Reply:
I took those out and realized that we still need to pull out the `completed` since the queue depends on that to decide whether or not to display an appointment. `notCompletedDescription` however, can be taken out.

Reply:
I mean should we even be passing back completed appointments at all? What's the point if it just hides it from the user anyway?